test_terms_fail.obo

```diff
@@ -0,0 +1,7 @@
+default-namespace: default_keyword_type
```

Comment:
This file keeps showing up in your commits for some reason. Can we try to remove this?